vrp-cli/src/lib.rs

```diff
@@ -525,9 +525,9 @@ pub fn get_solution_serialized(problem: Arc<CoreProblem>, config: Config) -> Res
 
     let mut writer = BufWriter::new(Vec::new());
     if let Some(metrics) = metrics {
-        (&solution, cost, &metrics).write_pragmatic_json(&problem, &mut writer)?;
+        (&solution, cost, &metrics).write_geo_json_and_pragmatic_json(&problem, &mut writer)?;
     } else {
-        (&solution, cost).write_pragmatic_json(&problem, &mut writer)?;
+        (&solution, cost).write_geo_json_and_pragmatic_json(&problem, &mut writer)?;
```

Comment:
This will change behavior of `solve_pragmatic` for every interop call and there will be no way to opt-out from geojson output which might be a problem in some cases.

I would propose two alternative ways to bring this behavior to python interop (and ideally, wasm + c): 

1. Introduce a new command, which will take problem, matrices and solution files in pragmatic formats and then run 
`serialize_solution_as_geojson(writer, problem, &geo_json_solution).map_err(|err| err.to_string())?;`
 **Pros**: 
     - easy way to opt-out
     - no breaking changes in public api
**Cons**:
    - efficiency: passing same json strings between FFI boundaries, deserialization costs

2. Modify `solve_pragmatic` command to take an additional argument which triggers desired behavior on demand
**Pros**:
    - easy to opt-out
    - no need to run additional command with its price
    - consistent with CLI which uses extra flag for json output in solve command
**Cons**:
    - small breaking change in public api which should be easy to adapt

I'm somehow in favor of the second approach

Reply:
I took the second approach, indeed. :smile: